README.md

```diff
@@ -94,3 +94,16 @@ export default loader({ error: 'errorMessage' })(MyComponent)
 // CustomErrorComponent will be displayed if 'error' prop is truthy
 export default loader({ ErrorIndicator: CustomErrorComponent })(MyComponent)
 ```
+
+### Delay parameter
+
+When a component loads very quickly, you will see a flash of the loading component.
+To avoid this behaviour, you can add a `delay` parameter to the loader with a time in milliseconds.
+Then, the loading indicator will be rendered, only after this delay (of course, if the component is loaded before, then it will be rendered)
```

Comment:
To much ponctuation for me.
Something like that maybe ?
`Then, the loading indicator will be rendered after the delay if the Component can't be rendered before that.`